pages/curriculum.tsx

```diff
@@ -77,6 +78,7 @@ const Curriculum: React.FC = () => {
             description={e.description}
             currentState={lessonState}
             reviewUrl={`https://c0d3.com/teacher/${e.id}`}
+            docUrl={e.docUrl}
```

Comment:
A good practice is to is to use `lesson` instead of `e` think about someone who's looking at  a piece of your code with a lot of `map()` he has no idea what mean `e`
eg:
<img width="559" alt="Screen Shot 2020-03-30 at 9 25 52 AM" src="https://user-images.githubusercontent.com/25457563/77938795-181f6800-726b-11ea-9f29-5d2bad6cfb25.png">